span_test.go

```diff
@@ -1,37 +1,37 @@
 package money
 
-import (
-	"testing"
-	"time"
+// import (
+// 	"testing"
+// 	"time"
 
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
+// 	"github.com/stretchr/testify/assert"
+// 	"github.com/stretchr/testify/require"
+// )
 
-func TestSpan(t *testing.T) {
-	t.Run("String", testSpanString)
-}
+// func TestSpan(t *testing.T) {
+// 	t.Run("String", testSpanString)
+// }
 
-func testSpanString(t *testing.T) {
-	startTime, e := time.Parse(time.RFC3339, "1970-01-01T00:00:01+00:00") //1 second into epoch time = 1,000,000 microseconds
-	require.Nil(t, e)
+// func testSpanString(t *testing.T) {
+// 	startTime, e := time.Parse(time.RFC3339, "1970-01-01T00:00:01+00:00") //1 second into epoch time = 1,000,000 microseconds
+// 	require.Nil(t, e)
 
-	i := &Span{
-		Name:    "test-span",
-		AppName: "test-app",
-		TC: &TraceContext{
-			TID: "test-trace",
-			SID: 1,
-			PID: 1,
-		},
-		Success:   true,
-		Code:      200,
-		StartTime: startTime,
-		Duration:  time.Second,
-		Host:      "localhost",
-	}
+// 	i := &Span{
+// 		Name:    "test-span",
+// 		AppName: "test-app",
+// 		TC: &TraceContext{
+// 			TID: "test-trace",
+// 			SID: 1,
+// 			PID: 1,
+// 		},
+// 		Success:   true,
+// 		Code:      200,
+// 		StartTime: startTime,
+// 		Duration:  time.Second,
+// 		Host:      "localhost",
+// 	}
 
-	var expected = "span-name=test-span;app-name=test-app;span-duration=1000000;span-success=true;span-id=1;trace-id=test-trace;parent-id=1;start-time=1000000;host=localhost;http-response-code=200"
+// 	var expected = "span-name=test-span;app-name=test-app;span-duration=1000000;span-success=true;span-id=1;trace-id=test-trace;parent-id=1;start-time=1000000;host=localhost;http-response-code=200"
 
-	assert.Equal(t, i.String(), expected)
-}
+// 	assert.Equal(t, i.String(), expected)
+// }
```

Comment:
Hi @johnabass maybe you would know why this test has been commented out? It seems this test works with the new changes. Your feedback would be greatly appreciated. TIA.

Reply:
I don't know why it was commented out.  If it's passing, and if the test case is still valid,  then I would uncomment it and go from there.

Reply:
OK, thanks for the feedback. I will re-enable it in a subsequent PR after this one has been merged.